test/runnable/cppa.d

```diff
@@ -1264,6 +1264,34 @@ void test16536()
     version(OSX) assert(pass16536(123) == 123);
 }
 
+/****************************************/
+// 15589 - extern(C++) virtual destructors are not put in vtbl[]
+
+extern(C++) {
```

Comment:
Curly braces on their own lines.

Reply:
Like the test immediately above... ;)

Reply:
Yes, like the rest of the file :smiley:.

Reply:
Except the one immediately above, which i used as reference..

Reply:
Ah, you mean the test for 15802? Half of that is using the correct style, the other half is not. Then there's the test for 16536 in between, as far as I can see.